docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+.. _supported_tasks:
+
+===================================
+Working with Labels and Predictions
+===================================
+
+Some checks, mainly the ones related to model evaluation, require labels and model predictions in order to run.
+In the deepchecks nlp package, predictions are passed into the suite / check ``run`` method as pre-computed
+predictions only (passing a fitted model is currently not supported).
+
+
+.. _supported_task__types:
+
+Supported Task Types
+====================
+
+Deepchecks currently supports two NLP task types:
+
+* **Text Classification**: Text classification is any NLP task in which a whole body of text (ranging from a sentence
+  to a document) is assigned a class (in the binary/multiclass case) or a certain set of classes (in the multilateral
+  case). In both the binary, the multiclass and the multilabel case the class "belongs" / "classifies" the whole text
+  sample. Examples for such tasks are Sentiment Analysis, Topic Extraction, detecting harmful content and more.
+* **Token Classification**: Token Classification is any NLP task in which each word (or to be more accurate, token) in
+  the text sample is assigned a class of it's own. In many cases most tokens will belong to a "background" class,
+  allowing the model to focus on the interesting tokens. Examples for such tasks are Named Entity Recognition, Part
+  of Speach annotation (in which all tokens have a non-background class).
+
+.. _supported_labels__predictions_format:
+
+Supported Labels and Predictions Format
+=======================================
+
+While labels are passed while constructing the :class:`TextData <text_data.TextData>` object, predictions are passed
+separately to the ``run`` method of the check / suite. Labels and predictions must be in the format detailed in this
+section, according the task type.
+
+Label Formats:
+--------------
+
+* **Text Classification** Label - For text classification the accepted label format differs between multilabel and
```

Comment:
I suggest to have the task type as an additional sub-title hierarchy...
(github markdown parses this incorrectly, so instead of text added picture)

<img width="220" alt="CleanShot 2022-09-12 at 15 37 12@2x" src="https://user-images.githubusercontent.com/33841818/189655303-88f53845-a87a-46e8-8845-8a2eba4dd1f3.png">